### DISCUSSION TITLE
Add option to reply to the comment in the Conversation tab

### DISCUSSION THREAD

**Discussion by samueldusek** [community/community#11674] [Pull Requests]
Sometimes I would like to reply to the comment of my colleague that he or she made to my pull request in the `Conversation` tab. It is very unpleasant experience to reply to the comment by making another comment and quote my colleague`s comment. 

It would be nicer to have an option to simply reply under his comment as it is when replying to comment that has been made to some file change in the `Files changed` tab.

I think it would make the conversation experience much better in the `Conversation` tab.

Thanks a lot for the response in advance. 🙂 

Big fun of GitHub. ❤️

**Comment by ML777AI** [comment]
Is it a bug?)

**Reply by samueldusek** [reply]
No, it is not a but. It something that is not possible right now and I believe it would be nice feature to add to improve the flow of the conversation in the Conversation tab when doing pull request review. 🙂
